.mvn/extensions.xml

```diff
@@ -2,6 +2,6 @@
   <extension>
     <groupId>io.jenkins.tools.incrementals</groupId>
     <artifactId>git-changelist-maven-extension</artifactId>
-    <version>1.0-beta-6</version>
+    <version>1.0-beta-7</version>
```

Comment:
Not actually necessary BTW. When there is a required update, you will know it because after updating the parent, `mvn validate` will fail.

Reply:
Just that it is not necessary. There were no changes since 1.0-beta-4.